docs/issuing-assets/control-asset-access.mdx

```diff
@@ -220,7 +220,7 @@ except BaseHorizonError as e:
 
 With that warning in mind:
 
-It is possible to lock down the issuing account of an asset so that the asset’s supply is permanently fixed. To do this, set the issuing account’s master weight to 0 using the Set Options operation. This prevents the issuing account from being able to sign transactions and therefore, is unable to issue any more assets. Be sure to do this only after you’ve issued all desired assets to the distribution account.
+It is possible to lock down the issuing account of an asset so that the asset’s supply cannot increase. To do this, first set the issuing account’s master weight to 0 using the Set Options operation. This prevents the issuing account from being able to sign transactions and therefore, making the issuer unable to issue any more assets. Be sure to do this only after you’ve issued all desired assets to the distribution account. Then make sure that the admin for the Stellar Asset Contract for this asset was not updated from the default (which is the issuer) using the `set_admin` contract call. If the admin was not the issuer, then the admin would be able to mint the asset even with the issuing account locked.
```

Comment:
Not every Stellar asset has a SAC, correct? Someone has to actually deploy one for that asset? Would it make sense to say something like: "If the asset has a SAC, make sure the admin for the SAC was not updated..."

Reply:
Yeah that's a good point. Fixed.